packages/cli/src/commands/list.ts

```diff
@@ -326,16 +322,23 @@ function getProjectName(d: Deployment) {
 
 // renders the state string
 export function stateString(s: string) {
-  switch (s) {
+  const CIRCLE = '● ';
+  // make `s` title case
+  s = `${s.substring(0, 1)}${s.toLowerCase().substring(1)}`;
+  switch (s.toUpperCase()) {
     case 'INITIALIZING':
-      return chalk.yellow(s);
-
+    case 'BUILDING':
+    case 'DEPLOYING':
+    case 'ANALYZING':
+      return chalk.yellow(CIRCLE) + s;
     case 'ERROR':
-      return chalk.red(s);
-
+      return chalk.red(CIRCLE) + s;
     case 'READY':
-      return s;
-
+      return chalk.green(CIRCLE) + s;
+    case 'QUEUED':
+      return chalk.white(CIRCLE) + s;
+    case 'CANCELED':
+      return chalk.gray(s);
     default:
       return chalk.gray('UNKNOWN');
```

Comment:
Do we want `UNKNOWN` to still be uppercase?

Reply:
I agree that either approach is fine. I wasn't sure if this was intentional or not.